lib/alert/addon/engine.js

```diff
@@ -0,0 +1,52 @@
+import Engine from 'ember-engines/engine';
+import loadInitializers from 'ember-load-initializers';
+import Resolver from './resolver';
+import config from './config/environment';
+
+const { modulePrefix } = config;
+
+const Eng = Engine.extend({
+  modulePrefix,
+  Resolver,
+  dependencies: {
+    services: [
+      'access',
+      'app',
+      'catalog',
```

Comment:
There are definitely services you don't actually need in here...

Reply:
Updated